templates/partials/pullquote.hbs

```diff
@@ -0,0 +1,13 @@
+<div class="flourish-pullquote-wrap">
+  <div class="flourish flourish-pullquote">
+     {{#each shapes}}
+      <div class="flourish-pullquote__{{shape}} {{#if reveal}}js-reveal-element{{/if}}" {{#if exclude}}data-exclude="{{exclude}}"
+        {{/if}}>
+        {{ flourishShapes shape }}
+      </div>
+    {{/each}}
+    <span class="flourish-pullquote__text">
```

Comment:
I'm really liking this much more now I've seen it on my laptop 👍I think it needs adjusting a bit on mobile though. This flourish seems too tall maybe? Could look better without the top padding too.

![image](https://user-images.githubusercontent.com/4202934/38873145-fe74c4e0-424c-11e8-9be2-e18784772daf.png)

Also the footer flourish looks a bit too abstract and could look better with little or no padding underneath it.

![image](https://user-images.githubusercontent.com/4202934/38873236-36c3217a-424d-11e8-8171-d6c0fc0621f6.png)

Reply:
Made these smaller and adjust the layout to reduce the weight on the page